src/charts/axis.js

```diff
@@ -226,11 +242,20 @@ export default function Axis (_container) {
     return this
   }
 
+  function unRotateXLabels () {
+    cache.xAxisRoot.select(".axis.x").selectAll("text")
+      .attr("x", 0)
+      .attr("y", 11)
```

Comment:
I wonder if these magic numbers could use the APPROX_FONT_WIDTH constant

Reply:
Perhaps, I think it comes from D3's internals?